<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>binaryheap_window</class>
 <widget class="QDialog" name="binaryheap_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_arbol">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>571</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_insertar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_makemin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_makemax">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Max</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_dato">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Dato</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_array">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_heapsort">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Heap Sort MIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_heap_sort_Max">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Heap Sort MAX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pop </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_pop">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Dato</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_test">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
